lib/components/label/label.less

```diff
@@ -96,6 +96,14 @@
     padding: 0 var(--su2); // Helps the label visually line up with inputs
 }
 
+// TODO: is this the right place for this?
```

Comment:
> I would like to get your feedback on the class naming s-required-symbol and location label.less.

I think this is a reasonable home for the required symbol styles. I _try_ to include each top-level component (i.e. any component that doesn't directly correspond to a parent component) in it's own less file, but I think this is a reasonable exception. If I consider where I'd think to look for this component, it would probably be here.

If for any reason we change our mind, it would be simple enough to relocate it, but I'm good with it here for now 🙂

---

> I did not prefix it with s-label since the class could be used elsewhere (e.g. in the legend explaining the abbreviation)

This also seems reasonable to me. No need to bind this directly to `.s-label`.